docs/services/postgresql/add-standby-node.md

```diff
@@ -32,6 +32,84 @@ $ cchq <env> ap setup_pg_standby.yml -e standby=[standby node]
 
 
 # Promoting a hot standby to Master
+1. Begin downtime for your site: `$ commcare-cloud <env> downtime start`
+
+2. Verify that the replication is up to date
+
+```
+$ commcare-cloud <env> run-shell-command pg1,pg2 'ps -ef | grep -E "sender|receiver"'
+
+    [ pg1 ] ps -ef | grep -E "sender|receiver"
+    postgres 5295 4517 0 Jul24 ? 00:00:01 postgres: wal sender process rep 10.116.175.107(49770) streaming 0/205B598
+
+    [ pg2 ] ps -ef | grep -E "sender|receiver"
+    postgres 3821 3808 0 Jul24 ? 00:01:27 postgres: wal receiver process streaming 0/205B598
+```
+
+Output shows that master and standby are up to date (both processing the same log).
+
+3. Promote the standby
+
 ```
-$ cchq <env> ansible-paybook promote_pg_standby.yml -e standy=[standby nde]
+$ commcare-cloud <env> ansible-paybook promote_pg_standby.yml -e standy=[standby node]
+```
+
+4. In your inventory remove `hot_standby_master` and `replication_slot` variables from your standby node, and remove the node from the `pg_standby` group.
+
+5. Update your processes to point to the newly promoted server:
+
+```
+$ commcare-cloud <env> update-config
+```
+
+6. If the standby you've promoted is one of the `form_processing` databases, update the PL proxy cluster
+
 ```
+$ commcare-cloud <env> django-manage --tmux configure_pl_proxy_cluster
+```
+
+7. If you have configured your standby and master nodes to use different parameters, or
+you would like to create replication slots on the newly promoted standby update those configurations:
+
+```
+$ commcare-cloud <env> ap deploy_db.yml --limit pg1,pg2
+```
+
+8. End downtime for your site: `$ commcare-cloud <env> downtime end`
+
+9. If you would like to have another standby for this newly promoted master, follow above instructions for adding a standby database.
+
+
+# Troubleshooting
+
+## Replication Delay
+https://www.enterprisedb.com/blog/monitoring-approach-streaming-replication-hot-standby-postgresql-93
+
+* Check if wal receiver and sender process are running respectively on standby and master using `ps aux | grep receiver` and `ps aux | grep sender`
+* Alternatively use SQL `select * from pg_stat_replication` on either master or standby
+* If WAL processes are not running, check logs, address any issues and may need to reload/restart postgres
+* Check logs for anything suspicious
+* Checking replication delay
+  * [Use datadog](https://app.datadoghq.com/dash/263336/postgres---overview?live=true&page=0&is_auto=false&from_ts=1511770050831&to_ts=1511773650831&tile_size=m&tpl_var_env=*&fullscreen=253462140&tpl_var_host=*)
```

Comment:
This is internal only, so you could describe what you would see here, or make a note that this won't work for people outside of our org.